### ISSUE TITLE
Make Dropdown component accessible

### ISSUE THREAD

**Issue by sydneywalcoff** [sydneywalcoff/app-tracker#163]
### Overview
Our dropdown component is currently not accessible to users navigating by keyboard or screen readers.

### Action Items
- [ ] make dropdown accessible

### Resources/Instructions
- [freecodecamp tutorial](https://www.freecodecamp.org/news/how-to-build-an-accessible-custom-dropdown-select-element/)
- [dev.to link](https://dev.to/ibrahimaq/how-to-create-a-custom-accessible-dropdown-with-react-and-typescript-2p3k)
